en/docs/develop-components/deploy-an-application-with-buildpacks.md

```diff
@@ -249,4 +249,13 @@ Follow the guidelines below based on your language:
     e.g.,
 
     `web: java -jar target/sample.war` <br>
-`
+
+### Build Time Environment Variables
+
+You can configure the environment variables that are necessary to build the component using the Build Configurations Editor in the component Build page. This option is not available for Ballerina and WSO2 MI.
+
+These environment variables and values will be passed to buildpack during the build time. Hence, this option can be used to configure buildpack specific environment variables or enviromemt variables required for component build.
```

Comment:
During the build process, the build-time environment variables and their values are passed to the buildpack. Therefore, you can configure both buildpack-specific environment variables and those required for the component build.